packages/core-components/src/components/Tabs/Tab.tsx

```diff
@@ -37,6 +37,8 @@ const tabMarginLeft = (isFirstNav: boolean, isFirstIndex: boolean) => {
   return '40px';
 };
 
+export type TabClassKey = 'root' | 'selected';
```

Comment:
Think you'll need a `/** @public */` here to get rid of the warning in the api-report.md

Reply:
done